packages/formdata-event/ts_src/formdata_listener_added.ts

```diff
@@ -167,19 +188,22 @@ export const wrapSubmitListener = (listener: EventListenerOrEventListenerObject)
     // event.
     if (getEventPropagationImmediatelyStopped(e) && targetIsAForm) {
       maybeDispatchFormdataForEvent(e);
-    // If the event's propagation was stopped (potentially before being seen by
-    // the capturing 'submit' listener) and this is the _last_ callback for its
-    // event phase, dispatch a 'formdata' event.
+      // If the event's propagation was stopped (potentially before being seen by
+      // the capturing 'submit' listener) and this is the _last_ callback for its
+      // event phase, dispatch a 'formdata' event.
     } else if (getEventPropagationStopped(e) && targetIsAForm) {
```

Comment:
```suggestion
    }
    // If the event's propagation was stopped (potentially before being seen by
    // the capturing 'submit' listener) and this is the _last_ callback for its
    // event phase, dispatch a 'formdata' event.
    else if (getEventPropagationStopped(e) && targetIsAForm) {
```

Maybe sticking the `}` before the comment would keep the indentation correct?